### ISSUE TITLE
BK-Abschluss Anlage D01 BiGa D3 / andere D01-Biga - welche Leistungseintragungen wo?

### ISSUE THREAD

**Issue by EwaldReinhardt** [SVWS-NRW/Schild3-BetaTest#1043]
In der Gliederung D01 wird zur Zeit angezeigt:
![grafik](https://github.com/SVWS-NRW/Schild3-BetaTest/assets/44644567/4fc30da9-2f9f-4bc4-a122-26f4258742ac)

Mir ist nicht klar was hier eingetragen werden soll.
Laut PO gibt es im BIGA Erzieher/AHR (Anlage D3):

> Berufsabschlussprüfung für Erzieherinnen und Erzieher
> Erste Teilprüfung
> Prüfungsfächer:
> 1. (schriftlich) Erziehungswissenschaften
> 2. (schriftlich) Biologie oder Deutsch
> 3. (schriftlich) Deutsch oder Englisch oder Religionslehre
> Zweite Teilprüfung
> 1. Projektarbeit
> 2. Fachpraktische Prüfung (Kolloquium)

In den anderen Bildungsgängen der Gliederung D01 (Berufsabschluss/AHR) Anlage D1, D2, D3a - D13):

> Berufsabschlussprüfung
> Erste Teilprüfung
> Prüfungsfächer:
> 1. (schriftlich) Elektrotechnik
> 2. (schriftlich) Mathematik
> 3. (mündlich) Gesellschaftslehre mit Geschichte oder Religionslehre
> 4. (schriftlich) Deutsch oder Englisch
> Zweite Teilprüfung
> Prüfungsfächer:
> 5. (schriftlich) Energie- oder Nachrichten- oder Datentechnik
> 6. (schriftlich) Wirtschaftslehre
> Praktische Prüfung   -> **in das Feld prakische Prüfung**

Was soll in das Feld Note Fachpraxis eingetragen werden? (Es gibt auch das Fach Fachpraxis in den Bildungsgängen!)
Hier bitte Hinweise geben was wo einzutragen ist und als Info im Programm verfügbar machen.

**Comment by AnneSchueller** [comment]
Hallo Ewald,
Die Details im BK-Abschluss wurden vor einiger Zeit von einem Fachberater erarbeitet, der selbst kein D01 an seiner Schule hat (soweit ich weiß)
Daher kann es natürlich sein, dass dieses Feld überflüssig ist. Die aktuellen Angaben wurden beim Switch auf SchILD3 unverändert übernommen. 
Wenn das nicht genutzt wird kann das m.E. auch weg.

**Comment by JuergenRichter** [comment]
Verstehe ich das richtig, dass für die D01 die Eingabe "Note Fachpraxis" überflüssig ist? Sind die anderen Angaben OK?

**Comment by EwaldReinhardt** [comment]
Ja, die Note Fachpraxis wird nicht benötigt bei den Assistenten (Gliederung D01 ohne Anlage D3)

Bei den Erziehern (Anlage D3 in der Gliederung D01) werden mehr Daten benötigt:
![grafik](https://github.com/SVWS-NRW/Schild3-BetaTest/assets/44644567/fa016f85-647c-4a64-a359-ed29d87da6da)
Hier sind auf dem Zeugnis auszugeben:
- Berufspraktische Leistung (einjähriges Berufspraktikum)
- Projektarbeit mit Thema
- Fachpraktische Prüfung (Kolloquium)
- Gesamtnote = "[2*Praktikum + Projekt + Kolloq]/4"
benötigt.

**Comment by AnneSchueller** [comment]
Aktuell kann für die Assistenten D01 nach wie vor im BK-Abschluss eine Note eingegeben werden.

Das war in Schild2 auch die letzten Jahre so. Daher wäre die Frage, ob hier Anpassungen notwendig sind, vor dem Hintergrund, dass die BKs im SVWS-Server neu organisiert werden sollen.

@EwaldReinhardt : Fehlen die von dir aufgelisteten Angaben für den Erzieher in Schild3?
@JuergenRichter : Wird denn die Darstellung im BK-Abschluss nur nach der Gliederung oder auch nach der Fachklasse unterschieden?

**Comment by AnneSchueller** [comment]
Gibt es eine andere Fachklasse, bei der das bereits so in Schild im Reiter BK-Abschluss implementiert ist?

**Comment by Raffenberg** [comment]
Ich schließe das Issue. Wenn das Problem noch besteht, bitte wieder eröffnen.
